webtask.json

```diff
@@ -31,6 +31,14 @@
       "default": "master",
       "required": true
     },
+    "GITHUB_HOST": {
+      "description": "Host of the GitHub instance (should be api.github.com for GitHub)",
+      "default": "api.github.com",
+      "required": true
```

Comment:
I would suggest making this an optional config, if left empty fall back to `api.github.com`.